sys/net/gnrc/application_layer/dhcpv6/client_simple_pd.c

```diff
@@ -84,15 +84,6 @@ static void _configure_upstream_netif(gnrc_netif_t *upstream_netif)
         addr.u8[15] = 2;
         gnrc_netif_ipv6_addr_add(upstream_netif, &addr, 64, 0);
     }
-
-    /* Disable router advertisements on upstream interface. With this, the border
-     * router
-     * 1. Does not confuse the upstream router to add the border router to its
-     *    default router list and
-     * 2. Solicits upstream Router Advertisements quicker to auto-configure its
-     *    upstream global address.
-     */
```

Comment:
Also: this comment still disappears.

Reply:
It disappears because RAs are no longer being disabled

Reply:
> I might move it to [`sys/include/net/gnrc/ipv6/nib/conf.h`](https://github.com/RIOT-OS/RIOT/pull/16590/files#diff-ee54067e04fea8cb5d587fbb1a5b9b3e342a486909e73b8bb2a3930bce1243afR132) though

Disagree the NIB has nothing to do with this particular use-case.